### DISCUSSION TITLE
add support for deleterr

### DISCUSSION THREAD

**Discussion by Niborian** [sct/overseerr#3761] [Feature Requests]
Add Support for deleterr under Services:

Create a platform or application that allows users to request services or items, like movies.
For each service (in this case, a movie), implement a "Request Deletion" feature.
Add a "Request Deletion" Button:

On the movie details or service page, add a "Request Deletion" button.
This button will allow users who have requested the service to initiate a deletion request.
Scenario 1: Multiple Requests and Deletion:

When multiple users request the same movie (e.g., Person 1 requests movie A and B, and Person 2 requests movie C):
Allow each user to request deletion for the movies they've requested.
Person 1 can request deletion for movie A and B but not for movie C.
"I Want This Too" Feature:

Implement an "I Want This Too" button for users who want the same service.
When Person 2 clicks "I Want This Too" on movie A, it indicates their interest in that movie.
Handling Deletion Requests:

When Person 1 requests deletion of movie A:
Check if there are other users (e.g., Person 2) who have also expressed interest in movie A.
If other users want the movie too (via "I Want This Too" button), do not delete it.
If no other users want it, proceed with the deletion request.
Result:

After Person 1's request, the system should show:
Person 1: Has requested movie B.
Person 2: Has requested (to stay) movie A.

This setup ensures that if there's continued interest in a movie, it won't be deleted, even if one user requests its deletion. It also allows for a collaborative approach to managing requested services.

**Comment by c11umw** [comment]
I like your idea of "I want this too" so people with an interest can get the same email notification rather than just the requestor.

For the other "keep/delete" bits, I've found users are either non-tech savvy enough or mostly just too lazy to manage the content they request.

So on our setup I use additional "temp" folders.
Any film in Temp >60 days gets auto purged
Any TV in Temp  > 90 days gets auto purged
(both via cron)

By default user requests go to these, but as admin I can override the destination for our own requests for to go directly into to the main library folders (no purge). In radarr/sonarr anything decent I can edit to the main folders too rather than temp.

My users know they normally have 2-3 months to watch what they requested and if it gets deleted before then they need to request it again. Everything else in our main personal library stays there - no purge.

Works pretty well that way so far. Everything is kept neat and users know what to do if their stuff has gone "missing"

Bonus is that I handbrake all the long term TV stuff to 720p and once transcoded I move the original 1080/2160p into the Temp TV folder. This way, if people watch it within the 90 days, then they can get the high-bitrate version while it's there.

**Reply by Niborian** [reply]
Well the reason I wnated the keep/delete thing is bc then I can setup overseerr to only allow 4 movie downloads and 1 tv-show, when they are done they can delete and then download (just need to be within the "quota") but I have also thougth about setting up a auto purge like you have.  😃

**Comment by immovably0** [comment]
Would it be possible to add a button allowing users to request the removal of content on Overserr? Some of them watch specific movies or series and would like the option to submit a removal request.
